### DISCUSSION TITLE
$_SERVER["REMOTE_ADDR"]

### DISCUSSION THREAD

**Discussion by KMaN1985** [chamilo/chamilo-lms#5801] [General]
I changed the server parameters. Currently, the system status in the "WEB Server" settings shows an incorrect remote address (previous) - $_SERVER["REMOTE_ADDR"]. How can I edit it to correct it?

**Comment by Kaneda-1** [comment]
Hello.
You will need to undo any changes you have made.
You cannot modify the Web server settings (Apache, Nginx, etc.) from Chamilo LMS.

**Reply by KMaN1985** [reply]
In Chamilo I updated the configuration.php file and nothing else. How to update Apache to provide the correct remote address in the $_SERVER table?
